ui/src/hooks/useMonitoring.js

```diff
@@ -33,7 +33,7 @@ export const useMonitoring = () => {
     (envName, routerName, revision) => {
       const clusterName = getEnvironmentCluster(envName, environments);
       const projectName = !!project ? project.name : undefined;
-
+      revision = !revision ? "$_all" : revision
```

Comment:
Perhaps this line can be absorbed into the function signature with a default argument?
```
const getMonitoringLink = (
  clusterName,
  projectName,
  routerName,
  routerVersion="$__all"
)
```

Reply:
Ok sure